### PR TITLE
Simple features w/ react

### PR DESCRIPTION
**Pull request by pdeleastar**
- New components added for overview, keystats and recentnews - (state stored in component). 
- Added a utility for async requests to the API

### CODE REVIEW

File: src/actions/actions.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+function news (data) {
+    return  {
+        type: NEWS,
+        data
+    }
+}
+
+export function handleNews (symbol) {
+    return (dispatch) => {
+      axios.get(`https://api.iextrading.com/1.0/stock/${symbol}/news/last/1`)
```
Comment (bhavesh-desai-scratch) [inline src/actions/actions.js]: No need for Axios. Just use native fetch.

File: src/components/KeyStats.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    componentDidMount() {
+        this.callOverview()
+    }
+
+    callOverview = async () => {
+        this.setState(() => ({
+            keystats: null
+        }));
+        const response = await fetchData(this.props.symbol, 'keystats');
```
Comment (bhavesh-desai-scratch) [inline src/components/KeyStats.jsx]: Please reframe from using async await for now

Comment (bhavesh-desai-scratch) [inline src/components/KeyStats.jsx]: Are you not using redux?

Comment (bhavesh-desai-scratch) [inline src/components/KeyStats.jsx]: Separate out data loading and presentation components.

Containers vs Components

File: src/actions/actions.js (hunk 1 above)
Comment (pdeleastar) [reply src/actions/actions.js]: Removed axios from project, now using native fetch
https://github.com/pdeleastar/stock-trader/pull/6/files

File: src/components/KeyStats.jsx (hunk 2 above)
Comment (pdeleastar) [reply src/components/KeyStats.jsx]: Removed async
https://github.com/pdeleastar/stock-trader/pull/6/files#diff-ad18ee503e34708bad505a743d444ae4

Comment (pdeleastar) [reply src/components/KeyStats.jsx]: Component was seperated out and made stateless
https://github.com/pdeleastar/stock-trader/pull/6/files#diff-ad18ee503e34708bad505a743d444ae4

**Pull request closed without merging**